EIPS/eip-5216.md

```diff
@@ -0,0 +1,116 @@
+---
+eip: 5216
+title: EIP-1155 Approval By Amount Extension
+description: Extension for EIP-1155 secure approvals
+author: Iván Mañús Murcia, Juan Carlos Cantó Martinez (@EscuelaCrypto in Telegram)
```

Comment:
This doesn't follow the format defined in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#eip-header-preamble).

We have a linting tool that'll pick up on common mistakes, and you can trigger it by closing and re-opening your PR, if you'd like.

Reply:
Hi Sam, thanks for your review. 

Do you mean the format of the authors? I don't see discrepancies in eip, title and description format.

Reply:
@SamWilsn Lies.  😬  It appears eipw failed to validate the author line for some reason.

@ivanmmurciaua See the author line in https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for description and example of what a proper authors line should look like.  Email or GitHub handles only (GitHub preferred for notification reasons and bot functionality).

Reply:
So, yes. I had a discrepancy with author line but is already fixed. Thanks both @SamWilsn @MicahZoltu